chronos/transports/stream.nim

```diff
@@ -1072,7 +1092,12 @@ when defined(windows):
                                              nil, flags)
           # Start tracking transport
           trackStream(ntransp)
-          server.createAcceptPipe()
+          try:
+            server.createAcceptPipe()
```

Comment:
trackStream(ntransp) should be inside try/except branch, so only successful attempts will be tracked.

Reply:
moved it after the try/catch which should have that same effect?